### ISSUE TITLE
New error in modelGetSignalsImages 

### ISSUE THREAD

**Issue by omidm7** [bahanonu/calciumImagingAnalysis#33]
Hey Biafra! Since pulling the newest updates in the master, I am running into a new error code after running cnmf-e. Everything seems to be fine until the very end, when I get this error:
![image](https://user-images.githubusercontent.com/44180252/66513122-3ad87d00-ea8f-11e9-89fb-0e60acd21e03.png)

... Which leads to this error:
![image](https://user-images.githubusercontent.com/44180252/66513202-665b6780-ea8f-11e9-8396-9433faeab3d2.png)

Do you know what could be causing this? I have attached the full command window output.
Omid
[Untitled 2.pdf](https://github.com/bahanonu/calciumImagingAnalysis/files/3709084/Untitled.2.pdf)

**Comment by bahanonu** [comment]
@omidm7 
Yep, seems `calciumImagingAnalysis.m` is missing that property on this repository (it's on my internal one).

Let me know if the pull request below fixes the issue. I'll do a full pull request with other updates later (mouse surgeries call).
https://github.com/bahanonu/calciumImagingAnalysis/pull/34

**Comment by omidm7** [comment]
Hey @bahanonu, this morning I'm getting the getSignalsImage error in both the master and #34!
Omid

**Comment by bahanonu** [comment]
@omidm7 
Do you mean the same error as before or a different one?

**Comment by omidm7** [comment]
Here is a screen shot of the error, it seems to be the getSignalsImage again:
![image](https://user-images.githubusercontent.com/44180252/66766549-2e736c00-ee63-11e9-9bc9-a1b46cf45e24.png)

I also attached the full output:
[Untitled 2.pdf](https://github.com/bahanonu/calciumImagingAnalysis/files/3725385/Untitled.2.pdf)

**Comment by bahanonu** [comment]
Does the regular expression for finding the movie match the movie name in the folder? e.g. highlighted text below:

![image](https://user-images.githubusercontent.com/5241605/66766843-f3256d00-ee63-11e9-82b3-00b45ae4425e.png)

I'll add a check for when the class can't find a movie and notify user to check regular expression then re-run.

**Comment by omidm7** [comment]
Ahh ok. The regEx was correct, but modelModifymovies had not finished correctly. Fixed it now, thanks!

**Comment by omidm7** [comment]
Hi Biafra, have you ever encountered this "licensing error"?

![image](https://user-images.githubusercontent.com/44180252/68242554-9aaa4100-ffc5-11e9-96bb-c636552924eb.png)
![image](https://user-images.githubusercontent.com/44180252/68242599-b1509800-ffc5-11e9-954f-bd56171a2957.png)

**Comment by bahanonu** [comment]
Only if a particular toolbox was not installed. If you type `license('test','Image_Toolbox')`, what is the output? Is your MATLAB from Stanford's CMGM (https://csbf.stanford.edu/software/)?

**Comment by omidm7** [comment]
Yeah, I'm using MATLAB from the CMGM.
The output for the line you gave me is ans = 1!

**Comment by bahanonu** [comment]
Strange, and you still get the license error for the image toolbox? If you try to use viewObjmaps or other functions besides CNMF-e, do you get errors?

**Comment by omidm7** [comment]
Yeah, when I try running modelVarsfromfiles or viewObjmaps, I get license errors for Signals_Toolbox and Statistics_Toolbox. I'm gonna try to re-activate the MathWorks license tomorrow!

**Comment by bahanonu** [comment]
Re-activating end up solving the issue?

**Comment by omidm7** [comment]
Apparently our lab's subscription to csbf had expired. I'm sure that's what was causing the licensing issue? I'm waiting for the admins to fix the subscription issue and I'll let you know!
Thanks Biafra,
Omid

**Comment by omidm7** [comment]
Hey Biafra, quick question:
After running cnmf-e , I get two cell maps. What is determining which cell is "accepted" at this point, before I've sorted using computeSortSignal? And is there I way I can sort through only the 1667 "accepted" cells rather than through the entire 2639 components?

![image](https://user-images.githubusercontent.com/44180252/69098846-4021d380-0a0e-11ea-8851-a9e68accb518.png)

**Comment by bahanonu** [comment]
Hey

It is determined by 
https://github.com/bahanonu/calciumImagingAnalysis/blob/master/image/filterImages.m
or if you have the class loaded, type `obj.filterImageOptions` for a list of some of the defaults that you can change.

What does figure 4001 look like? That seems like a large # of cells for a ~200x350 movie, there might be a large # of duplicates (e.g. need to change cnmf-e merging, etc. criteria).

cheers,
-Biafra

**Comment by omidm7** [comment]
Thanks! This is how 4001 looks. Changing the cnmf-e merging parameters helps reduce duplicates in some data sets, but not others. 

![image](https://user-images.githubusercontent.com/44180252/69109360-6191b800-0a2c-11ea-933f-610a908e8c13.png)

**Comment by bahanonu** [comment]
Thanks, what happens if you run PCA-ICA as a quick sanity check? 

Also, can you save the fig 46 and 4001 outputs as .fig files and attach them here so I can take a look?

**Comment by omidm7** [comment]
I can't attach .fig files here, so I'll email them to you right now.
